### PR TITLE
Updated README.md with a couple of things I ran across trying magellan

### PR DESCRIPTION
**Pull request by coridrew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,18 +54,18 @@ public class MainActivity extends SingleActivity {
...
     super.onCreate(savedInstanceState);
-    setContentView(R.layout.main_activity);
+    setContentView(R.layout.activity_main);
   }
 
 }
 ```
 
-`main_activity.xml`:
+`activity_main.xml`:
```
Comment (fabiendevos) [inline README.md]: why changing to activity_main.xml ?
main_activity is better english no? @ryanmoelter

File: README.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ Associated View `HomeView.java`:
 public class HomeView extends BaseScreenView<HomeScreen> {
   public HomeView(Context context) {
     super(context);
-    inflate(R.layout.home);
+    inflate(context, R.layout.home, this);
```
Comment (fabiendevos) [inline README.md]: the version without context will work as soon as we release the next version, but I agree it's confusing right now

File: README.md (hunk 1 above)
Comment (ryanmoelter) [reply README.md]: `activity_main.xml` is the Android convention and Android Studio's default naming for the `MainActivity`'s layout xml. I personally like `main_activity` better but that's not what most people use.

Comment (coridrew) [reply README.md]: I personally prefer main_activity as well, but like Ryan said, that's not the Android convention. I renamed it in the example solely to make it more understandable, across the board :)

Comment (coridrew) [reply README.md]: Although, it is nice to be able to alphabetically see these by groupings in a big project:
activity_
dialog_
fragment_
layout_
list_item_
view_

etc.

File: README.md (hunk 2 above)
Comment (coridrew) [reply README.md]: Ok, that's the only way I could get it to compile.

File: README.md (hunk 1 above)
Comment (fabiendevos) [reply README.md]: Ok, you two convinced me that it's easier to get started.

@coridrew : it's usually not optimal to group things in code based on type in my experience. :) A better grouping is usually to group by features: login_, profile_, settings_ etc. :)

**Comment by fabiendevos** [comment]
@coridrew thanks for contributing!

**Comment by coridrew** [comment]
Oh, I agree, for sure. My team generally groups by feature :)

**Pull request merged**